### PR TITLE
Made verified by field in consultation required

### LINKED ISSUE #4292: Make Verified By in Consultation Page as mandatory field
**Describe the bug**

Currently, the verified by field is an optional field in the consultation page

**To Reproduce**

https://care.coronasafe.in/facility/e02905de-2e0e-41ff-abd1-2eb169a509bc/patient/2db5d708-b2e0-4beb-9c55-fde58131ee6d/consultation/d6d2a4ef-a3cd-4f0a-9618-fa5a02878719/update

**Expected behavior**

Make the verified by field in the patient consultation page as mandatory 

**Screenshots**

image

### PR DESCRIPTION
**Pull request by skks1212**
## Proposed Changes

- Fixes #4292 
- Converted "Verified By" field to required

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 
image

I am able to submit the consultation form without filling the verified by data

**Comment by nihal467** [comment]
@skks1212 
image

https://deploy-preview-4295--care-net.netlify.app/facility/36f4f9e7-9e37-441f-904c-89562f24163f/patient/9adb2db4-3441-4b5e-aa93-8be0b1c75a29/consultation/6172aceb-95b9-4edc-abc1-d2c3c78962c4/update

iam still able to submit the consultation form without filling verified by column

**Comment by shivankacker** [comment]
@nihal467 seems to be working for me 🤔🤔
image

I added a few more checks, can you test again?

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -478,6 +478,19 @@ export const ConsultationForm = (props: any) => {
           }
           return;
         }
+
+        case "verified_by":
+          if (
+            !state.form[field] ||
+            state.form[field] === " " ||
+            !state.form[field].replace(/\s/g, "").length
```
Comment (khavinshankar) [inline src/Components/Facility/ConsultationForm.tsx]: @skks1212 do we need the 1st and 2nd condition, shouldn't this be covered in the 3rd condition?
```suggestion
            !state.form[field].replace(/\s/g, "").length
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**